src/lib/flexbox/api/layout-gap.ts

```diff
@@ -114,22 +116,23 @@ export class LayoutGapDirective extends BaseFxDirective implements AfterContentI
    * NOTE: this does NOT! differentiate between viewChildren and contentChildren
    */
   protected _watchContentChanges() {
-    let onMutationCallback = (mutations) => {
-      let validatedChanges = (it: MutationRecord) => {
-        return (it.addedNodes && it.addedNodes.length) ||
-            (it.removedNodes && it.removedNodes.length);
-      };
-
-      // update gap styles only for child 'added' or 'removed' events
-      if (mutations.filter(validatedChanges).length) {
-        this._updateWithValue();
+    this._zone.runOutsideAngular(() => {
+
+      if (typeof MutationObserver !== 'undefined') {
```

Comment:
Not a big deal, but you can move the `MutationObserver` support check outside of the `runOutsideAngular` to avoid hitting `runOutsideAngular` completely if it's unsupported.